sys/net/link_layer/ieee802154/Kconfig

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2019 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig KCONFIG_MODULE_IEEE802154
+    bool "IEEE802.15.4 configurations"
+    depends on MODULE_IEEE802154
+    help
+        Configure IEEE802.15.4 module using Kconfig
+
+if KCONFIG_MODULE_IEEE802154
+
+    config IEEE802154_DEFAULT_SUBGHZ_CHANNEL
+        int "IEEE802.15.4 default sub-GHZ channel"
+        default 5
+        range 0 11
+        help
+            IEEE802.15.4 default sub-GHZ channel
+            
+    config IEEE802154_DEFAULT_CHANNEL
+        int "IEEE802.15.4 default channel"
+        default 26
+        range 11 26
```

Comment:
same as for sub-GHz

![image](https://user-images.githubusercontent.com/1301112/74854201-5017bf00-533f-11ea-8c8d-ff5f30aa4ff7.png)

Reply:
done!